.github/ISSUE_TEMPLATE/bug_report_native.yml

```diff
@@ -0,0 +1,77 @@
+name: Bug Report for Quarkus native executables
+description: Report a bug in Quarkus that only appears when building a native executable
+labels: kind/bug
+body:
+  - type: textarea
+    id: description
+    validations:
+      required: true
+    attributes:
+      label: Describe the bug
+      description: >-
+        Describe the issue you are experiencing here to communicate to the
+        maintainers. Tell us what you were trying to do and what happened.
+
+        Make sure the issue is not reproducible in JVM mode. If the
+        issue is reproducible in JVM mode please open a "Bug Report for
```

Comment:
AFAIK there isn't a "Bug Report for Quarkus" issue type, just "Bug Report"

Reply:
Good catch. Fixed